src/ui/groups.ui

```diff
@@ -187,6 +177,56 @@
          </property>
         </widget>
        </item>
+       <item>
+        <spacer name="horizontalSpacer_2">
+         <property name="orientation">
+          <enum>Qt::Horizontal</enum>
+         </property>
+         <property name="sizeHint" stdset="0">
+          <size>
+           <width>40</width>
+           <height>20</height>
+          </size>
+         </property>
+        </spacer>
+       </item>
+      </layout>
+     </item>
+     <item>
+      <layout class="QHBoxLayout" name="horizontalLayout_2">
+       <item>
+        <widget class="QPushButton" name="autoGroups">
+         <property name="toolTip">
+          <string>Create language-based groups</string>
+         </property>
+         <property name="text">
+          <string>Auto groups by Languages</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QPushButton" name="autoGroupsFolders">
+         <property name="toolTip">
+          <string>Create language-based groups</string>
+         </property>
```

Comment:
folder?

Reply:
This is unrelated because I adjusted layout.

![image](https://user-images.githubusercontent.com/20123683/233523055-b810c3bc-0263-43ef-940f-9e29e32a5f77.png)

:sweat_smile:. This is why .ui file should be avoided. It doesn't work with git.

Reply:
Fixed